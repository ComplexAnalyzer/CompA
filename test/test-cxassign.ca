int main()
{ 

  cx a;
  a =(3.2,3.2);
  println(a);
  a[1]=3.4;
  a[0]= 24.3;
  println(a);
  return 0;
}
