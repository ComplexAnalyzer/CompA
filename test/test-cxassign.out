(3.200000,3.200000)
(24.300000,3.400000)